app/helpers/commodities_helper.rb

```diff
@@ -131,8 +131,10 @@ def divide_commodity_code(code)
   def abbreviate_commodity_code(commodity)
     code = commodity.code.to_s
 
-    return code if commodity.declarable?
+    commodity.declarable? ? code : abbreviate_code(code)
+  end
```

Comment:
I was tempted to move this method to the `Commodity`,
but I didn't, since  `abbreviate_code` must stay on the helper since it is used by
`SearchReference`.